packages/cheqd/src/anoncreds/services/CheqdAnonCredsRegistry.ts

```diff
@@ -308,8 +312,8 @@ export class CheqdAnonCredsRegistry implements AnonCredsRegistry {
 
       const revocationRegistryDefinitionResource = {
         id: utils.uuid(),
-        name: credentialDefinitionName as string,
-        resourceType: 'anonCredsRevocRegDef',
+        name: `${credentialDefinitionName}-${revocationRegistryDefinition.tag}`,
```

Comment:
we had issues with long names in the past, for which I think we came up with the solution to hash. Also, does this break already existing revocation registry definitions? Can we resolve both previously created ones, and the new one? And should the spec be updated?

Reply:
Right, as the credential definition name is hashed already in credo will this still be a problem?

No resolution is backward compatible. we are resolving using the resource id -> get the resource name from metadata to resolve the status lists.

Reply:
Okay thanks for the answers! I'm not sure, what is the max length of the tag? I think the credential definition name is max 32 because it's hashed. With the dash, the tag can max be 31 characters then. I think it should suffice, but if we really want it to be error prone we should maybe re-hash?

Reply:
Both the approaches sound good to me. Let me know if I should change it to the hashed one

Reply:
Yes let's do the hash. Or the cheqd spec should mention it can max be 31 characters long, but I think that's more prone to errors down the line. So hashing seems most stable to me

Reply:
This is complete